build.sbt

```diff
@@ -1,14 +1,33 @@
 import Dependencies._
+import sbt.Def
 
-val g8version = "0.7.3-SNAPSHOT"
+lazy val g8version = "0.7.3-SNAPSHOT"
+
+lazy val commonScriptedSettings = scriptedSettings ++ Seq(
+  scriptedLaunchOpts ++= sys.process.javaVmArguments.filter(
+    a => Seq("-Xmx", "-Xms", "-XX").exists(a.startsWith)
+  ),
+  scriptedBufferLog := false,
+  scriptedLaunchOpts += ("-Dplugin.version=" + version.value),
+  scripted := ScriptedPlugin.scripted.dependsOn(publishLocal in library).evaluated,
+  test in Test := scripted.toTask("").value
+)
+
+lazy val scala211libraryDependencies = Def.setting {
+  CrossVersion.partialVersion(scalaVersion.value) match {
+    case Some((2, scalaMajor)) if scalaMajor >= 11 => Seq(scalaXml, parserCombinator)
+    case _                                         => Nil
+  }
+}
 
 // posterous title needs to be giter8, so both app and root are named giter8
 lazy val root = (project in file("."))
-  .enablePlugins(TravisSitePlugin, NoPublish)
-  .aggregate(app, lib, scaffold, plugin)
+  .enablePlugins(NoPublish)
```

Comment:
Just cosmetic change. Plus it allows to comment / uncomment them while developing.